test/controllers/ConversationControllerSpec.scala

```diff
@@ -45,11 +45,11 @@ class ConversationControllerSpec extends PlaySpec with GuiceOneAppPerSuite with
       val messages = List(
         Message(
           SenderInformation("senderName", DateTime.parse("2021-02-19T10:29:47.275Z")),
-          Some(FirstReader("firstReadername", DateTime.parse("2021-03-01T10:29:47.275Z"))),
+          Some(FirstReader(DateTime.parse("2021-03-01T10:29:47.275Z"), Some("firstReadername"))),
```

Comment:
If FirstReader was a val in the test you would only have had to make one change.